xml/docker_concept.xml

```diff
@@ -0,0 +1,498 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xml:id="cha.docker.conceptoverview" xml:lang="en"
+ xmlns="http://docbook.org/ns/docbook" version="5.0"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>Conceptual Overview</title>
+ <para>
+  &deng; is a lightweight virtualization solution to run multiple virtual units
+  (containers) simultaneously on a single control host. Containers are isolated
+  with Kernel Control Groups (<xref linkend="vle.docker.cgroup"/>) and
+  <xref linkend="vle.docker.namespace"/>.
+ </para>
+ <para>
+  Full virtualization solutions such as &xen;, &kvm;, or &libvirt; are based on
+  the processor simulating a complete hardware environment and controlling the
+  virtual machines. However, &deng; only provides operating system-level
+  virtualization where the Linux kernel controls isolated containers.
+ </para>
+ <sect1 xml:id="sec.docker.installation.terminology_deng">
+  <title>&deng; Terminology</title>
+  <remark>toms 2019-02-04: check with Tanja, if you need the full title or
+  if "Terminology" is enough.</remark>
+  <para>
+   Before going into detail about &deng;, let us define some of the terms used:
+  </para>
+  <remark>toms 2019-02-04: two things:
+   1. Does it makes sense to sort the list alphabetically? It seems, we
+      discussed that already, but maybe ask Tanja what's her opinion on that.
+   2. I've asked Tanja, and we agreed, the term should NOT be repeated
+      in the definition. For example, in the "Registry" definition, you should
+      remove the "A registry is..." and write "Storage for already created
+      images." Same for the other terms.
+  </remark>
+
+  <variablelist>
+   <varlistentry xml:id="Docker_engine">
+    <term>&deng;</term>
+    <listitem>
+     <para>
+      &deng; is a server-client type application that performs all
+      tasks related to virtual machines. &deng; comprises the
+      following:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <formalpara>
+        <title>Daemon:</title>
+        <para>
+         The server side of &deng; manages all &docker; objects (images,
+         containers, network connections used by containers, etc.).
+        </para>
+       </formalpara>
+      </listitem>
+      <listitem>
+       <formalpara>
+        <title>REST API:</title>
+        <para>
+         Applications can use this API to communicate directly with the daemon.
```

Comment:
Use a similar structure like in the other explanations. As you want to define the term (what is the REST API?), I would rephrase to:
"An API which can be used to directly communicate with the daemon."